authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/integration/OTPManager.java

```diff
@@ -49,10 +49,10 @@
 public class OTPManager {
 
     /** The Constant QUERIED_STATUS_CODES. */
-    private static final List<String> QUERIED_STATUS_CODES = List.of(IdAuthCommonConstants.ACTIVE_STATUS, IdAuthCommonConstants.FROZEN);
+    public static final List<String> QUERIED_STATUS_CODES = List.of(IdAuthCommonConstants.ACTIVE_STATUS, IdAuthCommonConstants.FROZEN);
 
     /** The Constant OTP_EXPIRED. */
-    private static final String OTP_EXPIRED = "OTP_EXPIRED";
+    static final String OTP_EXPIRED = "OTP_EXPIRED";

```

Comment:
why access modifier changed from private to default?

Reply:
Corrected